src/app/help/help.component.html

```diff
@@ -0,0 +1,114 @@
+
+<div markdown ngPreserveWhitespaces>
+  # How to ...?
```

Comment:
How can we avoid the duplication between this an the How_To.md file?

Reply:
<!-- loaded from remote url -->
<markdown [src]="'path/to/file.md'" (load)="onLoad($event)" (error)="onError($event)"></markdown>
 
You can bind the file, but that was not working for me. Always have problem with the relative path.

Reply:
just used the wrong path :/. Now it works.

Reply:
awesome!

we still have two copies of the .md file in the repo, however. Can we let compodoc somehow grab it from the assets folder maybe?

Reply:
I think so. I just forgot the one in the doc folder